src/txn/txn.c

```diff
@@ -597,6 +597,141 @@ __wt_txn_release(WT_SESSION_IMPL *session)
     txn->operation_timeout_us = 0;
 }
 
+/*
+ * __txn_fixup_history_store --
+ *     Fix the history store update when a prepared transaction is updated.
+ */
+static int
+__txn_fixup_history_store(WT_SESSION_IMPL *session, WT_TXN_OP *op, WT_CURSOR *cursor, bool commit)
```

Comment:
Should the name of this function mention prepare?

Reply:
Fixed and updated the function comment also with more details. Can you check it once again?